### PR TITLE
Code of conduct revision

### PR DESCRIPTION
**Pull request by arokem**
Also includes proposed procedures for report and enforcement of the CoC.

Based on the following Google Doc: https://docs.google.com/document/d/1Cj9trFifGNqyNtpyEO9FfJLlmDGTbUy5enRAA9rTyH4/edit?tab=t.0#heading=h.214z66dv3mpo.

As this still includes portions that need to be filled out and some issues that need to be ironed out (e.g., retention of records about enforcement actions), this is still considered a draft.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,10 +1,83 @@
...
+
+[Report Documentation](#report-documentation)
+
+[Ongoing Documentation of Event and Community Bans](#ongoing-documentation-of-event-and-community-bans)
+
+[Privacy Concerns](#privacy-concerns)
+
+[Reference](#reference)
+
+## Code of Conduct {#coc}
```
Comment (effigies) [inline CODE_OF_CONDUCT.md]: Style point: These `{#anchor}`s are being shown but not linked. This does not seem to be a part of GitHub-flavored markdown.

There is an outline available on all Markdown documents by clicking the button in the top right of the page:

image

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -20,30 +93,295 @@ Examples of unacceptable behavior by participants include:
...
+
+The work group should not state who reported this incident. They should attempt to anonymize any identifying information from the report. The reported person should be discouraged from contacting the reporter to discuss the report. If they wish to apologize to the reporter, the work group can accept the apology on behalf of the reporter.
+
+### Decide further responses {#decide-further-responses}
+
+If the reported person provides additional context, the BIDS Code of Conduct work group may need to re-evaluate the behavioral modification plan and consequences.
+
+### Follow up with the reporter {#follow-up-with-the-reporter}
+
+A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are not made in good faith (such as "reverse sexism" or "reverse racism") may receive no response.
```
Comment (effigies) [inline CODE_OF_CONDUCT.md]:
```suggestion
A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.
```

I really don't think it's a good idea to pick two particular positions to represent bad faith reports. These phrases have particular meanings in American political discourse, but that meaning may not be shared by people of all international backgrounds or people who have not had their ear to the ground in the last decade or so.

I think stating that the group has discretion to determine bad faith or disparate impact ("Impact" section above) is sufficient to cover these cases without inviting debate over what these terms mean, which debate could itself turn into CoC violations.

**Review by effigies — APPROVED** [review]
Overall LGTM. One suggestion, though I'm not going to argue it further if others disagree.

**Comment by arokem** [comment]
Apologies, this has taken me an inexcusably long time to return to. 

I addressed comments/suggestions made by @effigies above (thanks!). 

@bids-standard/steering and @bids-standard/maintainers : any further comments? 

The one point that I think still needs to be addressed, before this can be enacted is the text regarding previous members of the steering group also being available to contact. On the one hand, I think that having previous steering group members available can be helpful. On the other hand, I don't want to assume that everyone who has been on SG is now willing to serve in this capacity. 

But: I also think that adding/removing names from this document is potentially an issue as well. So, it would be good if we had some way to point to people who are interested in serving in this capacity on some other page in a similar way to how we currently point to maintainers and SG in the section titled "General BIDS reporting procedure". However, I am not sure how we should do that. A separate section for "CoC Working Group" adjacent to these contact pages may be a potential way to go.

**Comment by effigies** [comment]
I think the response group is well-defined (3 people, at least one steering and one maintainer, unless conflicts of interest make this impossible), but the actions to be taken seem to flip between "working" and response groups, and it's a bit hard to follow.

Who is the working group? Is it just a synonym for response group, or does it mean the set of contactable people, i.e., current steering, current maintainers, and past steering who volunteer to remain in it?

**Comment by arokem** [comment]
Good point! I have clarified in the last change-set that "Working group" is maintainers + steering and "response group" is a smaller sub-set to evaluate a particular report. I think we'd also need to add some words [around here](https://bids.neuroimaging.io/collaboration/governance.html#other-working-groups)

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+
+In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+#### Report Handling Procedure
+
+When you make a report to a member of the Code of Conduct working group, they will gather information about the incident according to the BIDS Procedure For Incident Response.
+
+After an incident responder takes the report, they will immediately establish a 3 person Code of Conduct Response Group with at least 1 (past or present) BIDS steering group members and 1 BIDS maintainer, unless there is a conflict of interest, in which case any non-interested parties will be contacted.
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:95]: Reads oddly to me. Maybe this?

```suggestion
After an incident responder takes the report, they will immediately establish a 3 person Code of Conduct Response Group with at least 1 (past or present) BIDS steering group members and 1 BIDS maintainer, unless conflicts of interest prevent such an arrangement, in which case any disinterested party may be recruited.
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+
+If the incident is less urgent, the report will be discussed by the response group, which will meet to determine an appropriate response. Examples of possible incident responses are outlined in the BIDS Procedure For Incident Response.
+
+Procedures for in-person gatherings at e.g., conferences and workshops are likely handled by the local organizing committee to discuss all incidents and determine any necessary follow-up actions. Incidents can additionally fall within the scope of the BIDS CoC (e.g., if an incident occurs while a member of the community is speaking as a representative of BIDS).
+
+#### Following up with reporters
+
+Within one week of an incident report, a member of the response group, will follow up with the person who made the report and provided their contact information. The follow up may include:
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:105]:
```suggestion
Within one week of an incident report, a member of the response group will follow up with the reporter. The follow up may include:
```

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+
+* An acknowledgment that the Code of Conduct responders discussed the situation.
+* Whether or not the report was determined to be a violation of the Code of Conduct.
+* What actions (if any) were taken to address the reported behavior.
+
+In some cases, the lead responders may need to ask additional questions about the incident in order to identify the reported person.
+
+#### Conflict of Interest
+
+If an member of the response group has a conflict of interest for a report, they will recuse themselves from the discussion and handling of the incident. The incident documentation will not be available to them, and they will excuse themselves from any conversations involving handling the incident.
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:115]:
```suggestion
If a member of the working group has a conflict of interest for a report, they will recuse themselves from the discussion and handling of the incident. The incident documentation will not be available to them, and they will excuse themselves from any conversations involving handling the incident.
```

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+
+Decisions on the behavioral modification plans and consequences will be determined by a majority vote of the BIDS Code of Conduct response group.
+
+### Administrators/moderators approval
+
+Once the response group has approved the behavioral modification plans and consequences, they will communicate the recommended response to the BIDS maintainers. The response group does not need to state who reported this incident. They should attempt to anonymize any identifying information from the report.
+
+The BIDS CoC Working Group will be required to respond back with whether they accept the recommended response to the report. If they disagree with the recommended response, they should provide a detailed response or additional context as to why they disagree. The Working Group is encouraged to respond within a week.
+
+In cases where the Working Group disagrees on the suggested resolution for a report, the BIDS Code of Conduct work group may choose to notify the BIDS steering group.
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:219]: The working group includes the steering group.

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+
+### Follow up with the reported person
+
+The BIDS Code of Conduct work group will work with online community administrators/moderators to draft a response to the reported person. The email should contain:
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:223]:
```suggestion
The BIDS Code of Conduct response group will work with online community administrators/moderators to draft a response to the reported person. The email should contain:
```

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+* A description of the person's behavior in neutral language
+* The negative impact of that behavior
+* A concrete behavioral modification plan
+* Any consequences of their behavior
+
+The work group should not state who reported this incident. They should attempt to anonymize any identifying information from the report. The reported person should be discouraged from contacting the reporter to discuss the report. If they wish to apologize to the reporter, the work group can accept the apology on behalf of the reporter.
+
+### Decide further responses
+
+If the reported person provides additional context, the BIDS Code of Conduct work group may need to re-evaluate the behavioral modification plan and consequences.
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:234]:
```suggestion
If the reported person provides additional context, the BIDS Code of Conduct response group may need to re-evaluate the behavioral modification plan and consequences.
```

File: CODE_OF_CONDUCT.md [hunk 9]
```diff
@@ -20,30 +22,294 @@ Examples of unacceptable behavior by participants include:
...
+
+### Follow up with the reporter
+
+A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.
```
Comment (effigies) [inline CODE_OF_CONDUCT.md:238]:
```suggestion
A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the response group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.
```

**Review by effigies — COMMENTED** [review]
Minor cleanups.

The status of past steering members in the CoC WG is still a little unclear by my reading. I think it might make sense to formalize the CoC WG as consisting of: Current steering, maintainers, and past steering members who consent to remain on the WG. I think those past members should either be both potential initial contacts and response group candidates or else be neither.

Informally, I would suggest we refrain from involving past SG members unless they are the point of contact chosen by the reporter or there are difficulties pulling together a response group from active steering/maintainers.

To that effect, I think adding a section on the CoC WG to https://bids.neuroimaging.io/collaboration/governance.html would make sense. Until that's done, I think listing ex-steering group here is fine.
